demoinfocs_test.go

```diff
@@ -163,7 +163,7 @@ func TestDemoInfoCs(t *testing.T) {
 			t.Fatal(err)
 		}
 		if !ok {
-			t.Fatalf("Parser reported end of demo after less than %d frames", frameByFrameCount)
+			t.Fatalf("parser reported end of demo after less than %d frames", frameByFrameCount)
```

Comment:
Revert to upper-case

```suggestion
			t.Fatalf("Parser reported end of demo after less than %d frames", frameByFrameCount)
```